javascript-source/handlers/subscribeHandler.js

```diff
@@ -20,17 +20,21 @@
  *
  * Register new subscribers and unsubscribers in the channel
  */
-(function() {
+(function () {
     var subMessage = $.getSetIniDbString('subscribeHandler', 'subscribeMessage', '(name) just subscribed!'),
         primeSubMessage = $.getSetIniDbString('subscribeHandler', 'primeSubscribeMessage', '(name) just subscribed with Twitch Prime!'),
         reSubMessage = $.getSetIniDbString('subscribeHandler', 'reSubscribeMessage', '(name) just subscribed for (months) months in a row!'),
         giftSubMessage = $.getSetIniDbString('subscribeHandler', 'giftSubMessage', '(name) just gifted (recipient) a subscription!'),
+        giftAnonSubMessage = $.getSetIniDbString('subscribeHandler', 'giftAnonSubMessage', '(name) just gifted (recipient) a subscription!'),
         massGiftSubMessage = $.getSetIniDbString('subscribeHandler', 'massGiftSubMessage', '(name) just gifted (amount) subscriptions to random users in the channel!'),
+        massAnonGiftSubMessage = $.getSetIniDbString('subscribeHandler', 'massAnonGiftSubMessage', '(name) just gifted (amount) subscriptions to random users in the channel!'),
```

Comment:
Same message as Twitch here as well.

"An anonymous viewer gifted (amount) subscriptions to random viewers!"